assets/src/edit-story/components/panels/textStyle.js

```diff
@@ -89,8 +89,10 @@ function TextStylePanel({ selectedElements, onSetProperties }) {
   const fontWeights = getCommonValue(selectedElements, 'fontWeights');
   const fontStyle = getCommonValue(selectedElements, 'fontStyle') || 'normal';
   const fontFallback = getCommonValue(selectedElements, 'fontFallback');
-  const backgroundColor =
-    getCommonValue(selectedElements, 'backgroundColor') || '#000000';
+
+  // TODO make this work for multiple elements!
```

Comment:
This one is going to be a bit tricky. Or maybe not. Do a deep equal on all the selected elements' relevant property and see if they all match?

Reply:
We have a similar think with masks and also @miina is doing something similar to `flip`. My suggestion was to maybe enhance `getCommonValue` with a mapper function. E.g.

The old way is still good:
```
const commonFontStyle = getCommonValue(selectedEleemnts, 'fontStyle');
```

The new way is also good for more convoluted cases:
```
const commonColor = getCommonValue(selectedElements, ({color}) => somehowReturnANormalizedAndReusableValue(color));
```